cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/OperationInspectInfo.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.dataRecovery;
+import io.pravega.segmentstore.server.logs.operations.Operation;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class OperationInspectInfo extends Operation {
```

Comment:
This does not need to be a public class, right? Can you move it as an inner class in `DurableLogInspectCommand` please?